web/client-api/src/main/java/io/deephaven/web/client/api/widget/JsWidgetExportedObject.java

```diff
@@ -47,32 +85,47 @@ public TypedTicket typedTicket() {
         return typedTicket;
     }
 
+    /**
+     * Exports another copy of this reference, allowing it to be fetched separately. Results in rejection if the ticket
+     * was already closed (either by calling {@link #close()} or closing the object returned from {@link #fetch()}).
+     *
+     * @return a promise returning a reexported copy of this object, still referencing the same server-side object.
+     */
+    @JsMethod
+    public Promise<JsWidgetExportedObject> reexport() {
+        Ticket reexportedTicket = connection.getConfig().newTicket();
+        return Callbacks.grpcUnaryPromise(c -> {
+            ExportRequest req = new ExportRequest();
+            req.setSourceId(ticket.getTicket());
+            req.setResultId(reexportedTicket);
+            connection.sessionServiceClient().exportFromTicket(req, connection.metadata(), c::apply);
+        }).then(success -> {
```

Comment:
The downside of this is the extra round trip (not a `TicketAndPromise`), but it is a lot cleaner this way.

Reply:
Oh right - I can make it less clean and less round trippy....

Reply:
Hmm. I can, but then there will be zero round trips, and we would be ignoring failure (if any) from the exportFromTIcket call, and would only find out after trying to use the resulting object.

In that case, we could stop returning promise entirely.

Reply:
I'm indifferent. Could file a ticket to optimize this in the future?